.github/workflows/update.yml

```diff
@@ -0,0 +1,40 @@
+name: Update dist & source code
```

Comment:
I think it would be better to have `build` and `format` scripts as part of the `pre-commit` hook. Can you revert this change?

Reply:
I've added `husky` to take care of the build and the linting